src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue15559.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue15559 : _IssuesUITest
+	{
+		public override string Issue => "[iOS] Vertical layout content of label is truncated when a width request is set in the layout";
+
+		public Issue15559(TestDevice device) : base(device)
+		{ }
+
+		[Test]
+		[Category(UITestCategories.Layout)]
+		public void LabelDisplayWithoutCroppingInsideVerticalLayout()
```

Comment:
The test is failing on Android:
`Snapshot different than baseline: LabelDisplayWithoutCroppingInsideVerticalLayout.png (1.20% difference)`
In red the differences:
![image](https://github.com/user-attachments/assets/8dde439d-ed33-42ee-9a2e-a74478320ccf)

Could you review it?